src/main.ts

```diff
@@ -15,6 +15,11 @@ async function start() {
   const document = SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('api', app, document);
   app.useGlobalPipes(new ValidationPipe());
+  app.enableCors({
+    allowedHeaders: ['content-type'],
+    origin: 'http://localhost:3000',
```

Comment:
all links should be in env file. today yo have localhost tomorrow https://mysupercoolwesite.com

Reply:
Updated